app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmAuthor.java

```diff
@@ -15,47 +15,54 @@
  * @see Author
  */
 public class RealmAuthor extends RealmObject implements Author {
-    private String fullName;
-    private RealmAuthorContact authorContactInfo;
-    private RealmList<RealmPublication> publications;
-    private RealmList<RealmSeries> series;
-
-    /* Default constructor needed for Realm*/
-    public RealmAuthor(){}
-
-    public RealmAuthor(String fullName, RealmAuthorContact authorContactInfo,
-                       RealmList<RealmPublication> publications, RealmList<RealmSeries> series) {
-        this.fullName = fullName;
-        this.authorContactInfo = authorContactInfo;
-        this.publications = publications;
-        this.series = series;
-    }
-
-    /** Setters */
-    public void setFullName(String fullName){ this.fullName = fullName;}
-    public void setAuthorContactInfo(RealmAuthorContact authorContact){this.authorContactInfo = authorContact;}
-    public void setPublications(RealmList<RealmPublication> publications){
-        this.publications = publications;}
-    public void setSeries(RealmList<RealmSeries> series){
-        this.series = series;}
-    /** Getters */
-    @Override
-    public String getFullName() {
-        return this.fullName;
-    }
-
-    @Override
-    public RealmAuthorContact getAuthorContactInfo() {
-        return  this.authorContactInfo;
-    }
-
-    @Override
-    public RealmList<RealmPublication> getPublications() {
-        return  this.publications;
-    }
-
-    @Override
-    public RealmList<RealmSeries> getSeries() {
-        return  this.series;
-    }
+  private String fullName;
+  private RealmAuthorContact authorContactInfo;
+  private RealmList<RealmPublication> publications;
+  private RealmList<RealmStory> stories;
+
+  /* Default constructor needed for Realm*/
+  public RealmAuthor() {
+  }
+
+  public RealmAuthor(String fullName, RealmAuthorContact authorContactInfo,
+      RealmList<RealmPublication> publications, RealmList<RealmStory> stories) {
+    this.fullName = fullName;
+    this.authorContactInfo = authorContactInfo;
+    this.publications = publications;
+    this.stories = stories;
+  }
+
+  /** Setters */
```

Comment:
@grinnelldennis  the way we do it on the models side is 
if(input.isEmpty() || input == null)
don't set the value